partner-management-default.properties

```diff
@@ -201,7 +201,7 @@ mosip.iam.module.login_flow.scope=email
 mosip.iam.module.login_flow.response_type=code
 mosip.iam.authorization_endpoint=${keycloak.external.url}/auth/realms/mosip/protocol/openid-connect/auth
 mosip.iam.module.admin_realm_id=mosip
-mosip.pms.filtervalue.max_columns =100
+mosip.pms.filtervalue.max_columns=5000
```

Comment:
Can we make it 2000 instead?

Reply:
Yes, 2000 also fine, but we have a huge no of records for pms partner (now nearly 2000) anyhow it will increase that's why extended the limit as 5000